app/services/merchant_service.rb

```diff
@@ -0,0 +1,16 @@
+class MerchantService
+  def self.all_merchants
+    response = Faraday.get("http://localhost:3000/api/v1/merchants")
+    JSON.parse(response.body, symbolize_names: true)
+  end
+  
+  def self.singular_merchant(id)
+    response = Faraday.get("http://localhost:3000/api/v1/merchants/#{id}")
+    JSON.parse(response.body, symbolize_names: true)
+  end
+
+  def self.merchant_items(id)
+    response = Faraday.get("http://localhost:3000/api/v1/merchants/#{id}/items")
+    JSON.parse(response.body, symbolize_names: true)
+  end
+end
```

Comment:
Nice job with all of this. Since you are using this service to get data from three different endpoints, you may consider starting to abstract some of this code out to dry up your code. You could create a connection method as well as a method that is used to parse your response body.